engine/lib/views.php

```diff
@@ -1320,6 +1324,7 @@ function elgg_view_input($input_type, array $vars = array()) {
 		'id' => $id,
 		'input' => $input,
 		'class' => $field_class,
+		'input_type' => $input_type,
```

Comment:
Nothing currently, but can be helpful if someone wants to do stuff in
hooks, e.g. add field class to view vars
On Nov 3, 2015 7:04 PM, "Juho Jaakkola" <notifications@github.com> wrote:

> In engine/lib/views.php
> <https://github.com/Elgg/Elgg/pull/9106#discussion_r43783188>:
>
> > @@ -1320,6 +1324,7 @@ function elgg_view_input($input_type, array $vars = array()) {
> >  		'id' => $id,
> >  		'input' => $input,
> >  		'class' => $field_class,
> > +		'input_type' => $input_type,
>
> What is this used for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/9106/files#r43783188>.
>